<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugWindow</class>
 <widget class="QMainWindow" name="DebugWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1397</width>
    <height>767</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sinvi Drawer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="NameLabel">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckList" name="variables">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777201</height>
         </size>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="maxVisibleItems">
         <number>256</number>
        </property>
        <item>
         <property name="text">
          <string>new ...</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="OffseLabel">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Offset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="offset">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maximum">
         <number>99999</number>
        </property>
        <property name="displayIntegerBase">
         <number>16</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="TypeLabel">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="type">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="SizeLabel">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="size">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lenLabel">
        <property name="text">
         <string>Len</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="len"/>
      </item>
      <item>
       <widget class="QCheckBox" name="endian">
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>To Host Endian</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ShiftLabel">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Shift</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="shift">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maximum">
         <number>64</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="MaskLabel">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Mask</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="mask">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maxLength">
         <number>18</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Ratio</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="ratio">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="singleStep">
         <double>0.001000000000000</double>
        </property>
        <property name="stepType">
         <enum>QAbstractSpinBox::DefaultStepType</enum>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="applyButton">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="diversityMinLabel">
        <property name="text">
         <string>Diversity Min</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="diversityMin">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="valueMinLabel">
        <property name="text">
         <string>Min</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="minVal">
        <property name="minimum">
         <number>-2147483647</number>
        </property>
        <property name="maximum">
         <number>2147483647</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="valueMaxLabel">
        <property name="text">
         <string>Max</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="maxVal">
        <property name="minimum">
         <number>-2147483647</number>
        </property>
        <property name="maximum">
         <number>2147483647</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ampMinLabel">
        <property name="text">
         <string>Amplitude Min</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="ampMin">
        <property name="maximum">
         <number>2147483647</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ampMaxLabel">
        <property name="text">
         <string>Amplitude Max</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="ampMax">
        <property name="maximum">
         <number>2147483647</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="autoSearch">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="ChartView" name="chart">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="renderHints">
       <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QTimeEdit" name="timeFrom">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="displayFormat">
         <string>hh:mm:ss</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QSlider" name="sliderFrom">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sliderTo">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="invertedControls">
           <bool>true</bool>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTimeEdit" name="timeTo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="displayFormat">
         <string>hh:mm:ss</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1397</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenPcap"/>
    <addaction name="actionSpy_Device"/>
    <addaction name="actionOpen_UDP_Socket"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Configuration"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionFix_chart_range"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionLicence"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionStream_to_external"/>
    <addaction name="actionServer_mode"/>
    <addaction name="actionSet_PCAP_filter"/>
    <addaction name="actionSelect_network_device"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpenPcap">
   <property name="text">
    <string>Open PCAP ...</string>
   </property>
  </action>
  <action name="actionSpy_Device">
   <property name="text">
    <string>Spy Device ...</string>
   </property>
  </action>
  <action name="actionOpen_UDP_Socket">
   <property name="text">
    <string>Connect to UDP ...</string>
   </property>
  </action>
  <action name="actionSave_Configuration">
   <property name="text">
    <string>Save Configuration</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export ...</string>
   </property>
  </action>
  <action name="actionFix_chart_range">
   <property name="text">
    <string>Fix chart range</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionLicence">
   <property name="text">
    <string>Licence</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionStream_to_external">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Stream to external</string>
   </property>
  </action>
  <action name="actionServer_mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Server mode</string>
   </property>
  </action>
  <action name="actionSet_PCAP_filter">
   <property name="text">
    <string>Set PCAP filter</string>
   </property>
  </action>
  <action name="actionSelect_network_device">
   <property name="text">
    <string>Select network device</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">chartview.h</header>
  </customwidget>
  <customwidget>
   <class>QCheckList</class>
   <extends>QComboBox</extends>
   <header>include/checklist.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
